packages/teams-js/src/private/nestedAppAuth/nestedAppAuthBridge.ts

```diff
@@ -260,6 +264,7 @@ function createNestedAppAuthRequest(data: string): NestedAppAuthRequest {
     uuid: generateUUID(),
     func: 'nestedAppAuth.execute',
     timestamp: timestamp,
+    apiVersionTag: 'v2_nestedAppAuth.execute',
```

Comment:
Could you please comment the reason for using hardcoded string here. We do not recommend having hardcoded string for `apiVersionTag` but in your case, you have special reason to do so.

Reply:
Thanks. I added comment. Let me know if you have any other concerns.